iOS App build fails

**Overview (Required)**
- iOS App のビルドに失敗します

**Steps To Reproduce**

1. Clone the repository. commit: https://github.com/DroidKaigi/conference-app-2020/commit/a3be5cb767dd6383c0e830d92f1cb3704088f4dd
1. Build Android app with Android Studio 3.6. (This step may not be necessary.)
1. Run `make init` in the iOS App directory and open the workspace.
```
cd ios-base && make init && open DroidKaigi\ 2020.xcworkspace
```
4. Run `Build` 
5. See error
```
> Configure project :ios-combined
Kotlin Multiplatform Projects are an experimental feature.
FAILURE: Build failed with an exception.
* What went wrong:
Task ‘’ not found in project ‘:ios-combined’.
* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
BUILD FAILED in 2s
mkdir: /Users/nori/work/conference-app-2020/ios-base/build: File exists
cp: /Users/nori/work/conference-app-2020/ios-base/../ios-combined/build/bin/iosX64/debugFramework/ios_combined.framework: No such file or directory
Command PhaseScriptExecution failed with a nonzero exit code
```

**Env:**
 - Xcode 11.3 

**Additional context**

--- a/ios-base/Configs/DroidKaigi 2020-Debug.xcconfig
+++ b/ios-base/Configs/DroidKaigi 2020-Debug.xcconfig
@@ -1,6 +1,7 @@
 // Generated using xcconfig-extractor 0.6.0 by Toshihiro Suzuki - https://github.com/toshi0383/xcconfig-extractor
 #include "DroidKaigi 2020.xcconfig"
 #include "Pods/Target Support Files/Pods-DroidKaigi 2020/Pods-DroidKaigi 2020.debug.xcconfig"
+KN_LIBRARY_BUILD_TASK[sdk=iphonesimulator*] = linkIosX64
+KN_LIBRARY_BUILD_TASK[sdk=iphoneos*] = linkIosArm64
 KN_LIBRARY_BUILD_PATH[sdk=iphonesimulator*] = $SRCROOT/../ios-combined/build/bin/iosX64/debugFramework/ios_combined.framework
 KN_LIBRARY_BUILD_PATH[sdk=iphoneos*] = $SRCROOT/../ios-combined/build/bin/iosArm64/debugFramework/ios_combined.framework
-

--- a/ios-base/Configs/DroidKaigi 2020-Release.xcconfig
+++ b/ios-base/Configs/DroidKaigi 2020-Release.xcconfig
@@ -1,6 +1,7 @@
 // Generated using xcconfig-extractor 0.6.0 by Toshihiro Suzuki - https://github.com/toshi0383/xcconfig-extractor
 #include "DroidKaigi 2020.xcconfig"
 #include "Pods/Target Support Files/Pods-DroidKaigi 2020/Pods-DroidKaigi 2020.release.xcconfig"
-KN_LIBRARY_BUILD_PATH[sdk=iphoneos*] = $SRCROOT/../ios-combined/build/bin/iosArm64/releaseFramework/ios_combined.framework
+KN_LIBRARY_BUILD_TASK[sdk=iphonesimulator*] = linkIosX64
+KN_LIBRARY_BUILD_TASK[sdk=iphoneos*] = linkIosArm64
 KN_LIBRARY_BUILD_PATH[sdk=iphonesimulator*] = $SRCROOT/../ios-combined/build/bin/iosX64/releaseFramework/ios_combined.framework
-
+KN_LIBRARY_BUILD_PATH[sdk=iphoneos*] = $SRCROOT/../ios-combined/build/bin/iosArm64/releaseFramework/ios_combined.framework

